day_generator.dart

```diff
@@ -105,3 +115,69 @@ class Day$dayNumber extends GenericDay {
 
 ''';
 }
+
+String _testTemplate(String day) {
+  return '''
+import 'package:test/test.dart';
+
+import '../solutions/day$day.dart';
+
+// *******************************************************************
+// Fill out the variables below according to the puzzle description!
+// The test code should usually not need to be changed.
+// *******************************************************************
+
+/// Paste in the small example that is given for the FIRST PART of the puzzle.
+/// It will be evaluated again the `_exampleSolutionPart1` below.
+const _exampleInput1 = \'''
+\''';
+
+/// Paste in the small example that is given for the SECOND PART of the puzzle.
+/// It will be evaluated against the `_exampleSolutionPart2` below.
+///
+/// In case the second part uses the same example, uncomment below line instead:
+// const _exampleInput2 = _exampleInput;
+const _exampleInput2 = \'''
+\''';
+
+/// The solution for the FIRST PART's example, which is given by the puzzle.
+const _exampleSolutionPart1 = 0;
```

Comment:
This would indeed pass at the start, which might not be great from a purist POV. 
But as this will not pass as soon as the user adds the actual solution from the example (or adds his solution) - and I expect them to do this when they use the test - I think it is ok.
An even bigger case for me to leave it as is: The examplePart2 will not be distracting/annoying (red 😅) until you acutally start to work at it :)